eo-runtime/src/test/java/org/eolang/UniverseDefaultTest.java

```diff
@@ -146,6 +147,7 @@ void copies() {
     }
 
     @Test
+    @Disabled
```

Comment:
@maxonfjvipon no puzzle for this?

Reply:
@yegor256 there's one [here](https://github.com/objectionary/eo/pull/2583/files#r1392404872)